get_dependencies.py

```diff
@@ -3,15 +3,33 @@
 import tarfile
 from zipfile import ZipFile
 import requests
+from rfc6266 import parse_requests_response
+import posixpath
 
 
-def download(url, filename):
+def download(url):
+    response = requests.get(url, stream=True)
+    filename = parse_requests_response(response).filename_unsafe
+
+    default_filename = 'file'
+    if filename is None:
```

Comment:
As I said, if we don't find the filename in the Content-Disposition header or as the basename of the URL, we want to throw an exception.